styles.css

```diff
@@ -1,9 +1,67 @@
-.title{
-  font-family:'Roboto', sans-serif;
+body {
+  font-family: 'Roboto', sans-serif;
+  margin: 0; }
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6195316)